bundles/AdminBundle/public/js/pimcore/layout/toolbar.js

```diff
@@ -410,7 +410,18 @@
                      }
                  });
              }
- 
+
+             this.extrasMenu = new Ext.menu.Menu({
```

Comment:
TODO: Please adapt accordingly the toolbar to the latest implementation that are in the glossary bundle
See also https://github.com/pimcore/pimcore/pull/13851#discussion_r1058878772